src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerTraitTest.php

```diff
@@ -518,7 +533,7 @@ public function testCreateFormBuilder()
 
     public function testGetDoctrine()
     {
-        $doctrine = $this->getMockBuilder(ManagerRegistry::class)->getMock();
+        $doctrine = $this->createMock(ManagerRegistry::class);
```

Comment:
@nicolas-grekas @derrabus any idea why this test is failing with PHP 7.4? 🤔

Reply:
`createMock` adds calls `disallowMockingUnknownTypes` on the mockBuilder?

Reply:
Yes, but shouldn't we fix this somehow or mark it as skipped?

Reply:
see #39968